opendatahub-io/odh-dashboard
# Fix bug in migration for GPUS

closes: https://github.com/opendatahub-io/odh-dashboard/issues/1678

## Description



This PR is to remove auto migration for notebook containers and add a solution for containers that are using gpus but not profiles

## How Has This Been Tested?




1. start up main branch on a GPU cluster and create a workbench with a GPU
2. stop backend
3. switch to accelerator-support branch and start backend
4. console should say migration was successful
5. Workbench should have the keep what i have option selected

## Test Impact


bug fix

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
